### PR TITLE
Events tool with GraphQL and Events resource completion

### PR DESCRIPTION
**Pull request by ClayMav**
- [x] Cleanup extraneous code
- [x] Ensure no extraneous packages
- [x] Add types to sensible things and props
- [x] Remove buttons that will not be launched yet (advert and qr)

### CODE REVIEW

File: api/src/resources/Event/entity.ts [hunk 1]
```diff
@@ -38,19 +37,20 @@ export class Event extends BaseEntity {
...
-  @OneToMany((type: void) => Sig, (sig: Sig) => sig.hostedEvents, {
-    lazy: true
+  @Field((returns: void) => Sig)
+  @ManyToOne((returns: void) => Sig, (sig: Sig) => sig.hostedEvents, {
+    lazy: true,
+    nullable: false
   })
   @JoinTable()
-  public hostSigs: Lazy<Sig[]>;
+  public hostSig: Lazy<Sig>;
```
[the remark this answers is not in the document]
Comment (ClayMav) [reply api/src/resources/Event/entity.ts]: Visually it will be difficult to add to our website.

[the remark this answers is not in the document]
Comment (ClayMav) [reply api/src/resources/Event/entity.ts]: In a way that is pleasing and non-destructive UX

**Comment by ClayMav** [comment]
I have decided not to tackle the other things you mentioned

File: apps/admin-web/src/index.tsx [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+  activeEvent: undefined
+});
+
+const httpLink: any = createHttpLink({
+  uri: config.GRAPHQL_URL
+});
+
+const authLink: any = setContext((_: any, { headers }: any) => {
+  // get the authentication token from local storage if it exists
+  const token: string | null = localStorage.getItem("token");
```
Comment (kevinschoonover) [inline apps/admin-web/src/index.tsx]: Can you store this name in an config variable just in case other things need to grab it.

**Pull request merged**